app.js

```diff
@@ -20,7 +20,7 @@ const postsRoute = require("./routes/postings");
 const usersRoute = require("./routes/users");
 
 app.use(cors());
-app.use(bodyParser.json());
+app.use(bodyParser.json({limit: '50mb'}));
```

Comment:
Increase request limit to accommodate image strings

Reply:
suffices to let users know the max supported image size/ resolution on the frontend if its not already added?

Reply:
I am still trying to figure out how to limit the size and number of images uploaded by the user. It will ideally be included in the frontend. For now, I will add an instruction in the UI which they will hopefully follow